modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldTemplateContextContributor.java

```diff
@@ -54,12 +62,42 @@
 			"showMaximumRepetitionsInfo",
 			GetterUtil.getBoolean(
 				ddmFormField.getProperty("showMaximumRepetitionsInfo"))
+		).put(
+			"systemSettingsURL",
+			() -> {
+				if (!GetterUtil.getBoolean(
+						ddmFormField.getProperty(
+							"showMaximumRepetitionsInfo"))) {
+
+					return StringPool.BLANK;
+				}
+
+				return getSystemSettingsURL(
+					ddmFormFieldRenderingContext.getHttpServletRequest());
+			}
 		).put(
 			"value",
 			GetterUtil.getBoolean(ddmFormFieldRenderingContext.getValue())
 		).build();
 	}
 
+	public String getSystemSettingsURL(HttpServletRequest httpServletRequest) {
```

Comment:
@carolmariaabb, does it need to be `public` or can we make it `protected`/`private`? If it doesn't need to be `public`, can you please change it?

Reply:
@natocesarrego I changed it to private. Thanks, I hadn't noticed it was public.